extension/src/cli/RefreshXLF.ts

```diff
@@ -0,0 +1,108 @@
+import * as path from "path";
+import * as fs from "fs";
+import { LanguageFunctionsSettings } from "../Settings/LanguageFunctionsSettings";
+import {
+  _refreshXlfFilesFromGXlf,
+  updateGXlfFromAlFiles,
+} from "../XliffFunctions";
+import * as WorkspaceFunction from "../WorkspaceFunctions";
+import * as CliSettingsLoader from "../Settings/CliSettingsLoader";
+import { logger, setLogger } from "../Logging/LogHelper";
+import { ConsoleLogger } from "../Logging/ConsoleLogger";
+
+setLogger(new ConsoleLogger());
+const functionName = "RefreshXLF.js";
+
+enum Option {
+  updateGxlf = "--update-g-xlf",
+  failOnChange = "--fail-changed",
+}
+interface Parameters {
+  workspaceFolderPath: string;
+  updateGxlf: boolean;
+  failOnChange: boolean;
+}
+
+const usage = `
+Usage:
+$> node ${functionName} <path-to-al-app-folder> [${Object.values(Option).join(
```

Comment:
@theschitz I think we need the workspaceFilePath as well (as in CreateDocumentation.js)
We read the settings from workspaceFilePath and overrides from the settings.json in the .vscode folder.